src/PermitOptions.cs

```diff
@@ -8,43 +8,17 @@ internal record PermitServiceOptions(PermitOptions Options, PermitProvidersOptio
 /// <summary>
 /// Permit SDK options
 /// </summary>
+[Serializable]
 [ExcludeFromCodeCoverage]
 public class PermitOptions
 {
     /// <summary>
     /// Token to access Permit APIs
     /// </summary>
     public string? ApiKey { get; set; }
+    
     /// <summary>
     /// The URL of the PDP
     /// </summary>
     public string PDP { get; set; } = "http://localhost:7766";
-    /// <summary>
-    /// Default tenant
-    /// </summary>
-    public string DefaultTenant { get; set; } = "default";
```

Comment:
Why did you remove those options, how can the user set it up?

Reply:
Especially DebugMode and ApiUrl

Reply:
So the reason is, the current SDK is used to call both the PDP and the Permit API. This one is just for the PDP so The ApiUrl is not needed. For the debug mode, I dont know what it does in the SDK because the PDP debug flag is set on Docker not in SDK. The other options are more or less the same. 

Default tenant can be set with the AspNet specific options, in like 5 ways, but I can add a global static option which might be worth it

Reply:
I added the tenant options back. 

DebugMode does this in the .NET SDK. I can add it back but I'd prefer to use _logger.LogTrace or LogDebug, instead of using Info for everything

```
this.logger.LogInformation(
                            string.Format(
                                "Checking permission for {0} to perform {1} on {2}",
                                user,
                                action,
                                resource.type
                            )
                        );
                        
```